R/ranks_multinom.R

```diff
@@ -1,20 +1,25 @@
-
 #' Confidence sets for ranks based on multinomial data
 #'
+#' Given data with counts of successes for each category, calculate confidence set for ranks by most successes for categories.
```

Comment:
change to: "Given data on counts of successes for each category, calculate confidence sets for the ranks of categories, where categories are ranked by their success probabilities."